### ISSUE TITLE
could not attach to pid : lstat /my_service_name: no such file or directory

### ISSUE THREAD

**Issue by gopiextrahop** [go-delve/delve#2174]
Please answer the following before submitting your issue:

Note: Please include any substantial examples (debug session output,
stacktraces, etc) as linked gists.

1. What version of Delve are you using (`dlv version`)?
```
Delve Debugger
Version: 1.5.0
Build: $Id: ca5318932770ca063fc9885b4764c30bfaf8a199 $
```

2. What version of Go are you using? (`go version`)?
```
go version go1.14.2 linux/amd64
```
3. What operating system and processor architecture are you using?
```
$ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
VERSION_CODENAME=stretch
ID=debian
```

4. What did you do?
Tried to attach `dlv` to a PID.

5. What did you expect to see?
Attaching `dlv` to PID should succeed.

6. What did you see instead?
Attaching `dlv` to PID  fails. 

* This works fine in v1.2.0, v1.3.2 and v1.4.0. 
* Seems broken in v1.4.1 and v1.5.0. 

```bash
# This is a kubernetes pod/docker container process id.
sudo ./dlv attach 25031
could not attach to pid 25031: lstat /my_service_name: no such file or directory
```
* This sees similar to [bug](https://github.com/go-delve/delve/issues/1698) which is closed, but I don't see it is fixed.

**Comment by aarzilli** [comment]
What happens if you do `ls -l /my_service_name`?

**Comment by gopiextrahop** [comment]
```bash
gopi@ip-xxx:~$ ps -eo pid,lstart,cmd | grep my_service_name | grep -v grep
 3778 Sun Sep 13 05:34:08 2020 /my_service_name
 
gopi@ip-xxx:~$ sudo ls -l /my_service_name
ls: cannot access '/my_service_name': No such file or directory

gopi@ip-xxx:~$ sudo stat /proc/3778/exe
  File: /proc/3778/exe -> /my_service_name
  Size: 0               Blocks: 0          IO Block: 1024   symbolic link
Device: 4h/4d   Inode: 561940234   Links: 1
Access: (0777/lrwxrwxrwx)  Uid: (419724/ UNKNOWN)   Gid: (    0/    root)
Access: 2020-09-13 19:04:31.551996361 +0000
Modify: 2020-09-13 19:04:31.551996361 +0000
Change: 2020-09-13 19:04:31.551996361 +0000
 Birth: -
 
gopi@ip-xxx:~$ sudo ./dlv attach 3778
 could not attach to pid 3778: lstat /my_service_name: no such file or directory
```

**Comment by aarzilli** [comment]
Duplicate of #2168, see PR.

**Comment by aarzilli** [comment]
Closing, duplicate of #2168, fixed by 85952c08267be75f85cf3ad92254cd57bebd8dc2
